plugins/inputs/suricata/suricata_test.go

```diff
@@ -297,85 +237,23 @@ func TestSuricataEmptyJSON(t *testing.T) {
 		},
 	}
 	acc := testutil.Accumulator{}
-	acc.SetDebug(true)
-
-	assert.NoError(t, s.Start(&acc))
+	require.NoError(t, s.Start(&acc))
+	defer s.Stop()
 
 	c, err := net.Dial("unix", tmpfn)
 	if err != nil {
 		log.Println(err)
+
 	}
 	c.Write([]byte("\n"))
 	c.Close()
 
 	acc.WaitError(1)
-
-	s.Stop()
-}
-
-func TestSuricataInvalidInputs(t *testing.T) {
```

Comment:
Was this test removed because it's not useful?  If so, we also need to remove the unused brokenType* and brokenStruct* vars at the top

Reply:
These tests were removed to avoid testing log output.  The test was inherently racy; there is no way to know when the logging has completed as there is no feedback to the sender.  Time permitting, I'd like to come back to this plugin since I believe some of these warnings ought to be errors that could be tested.

Reply:
I have removed the now unused variables.